postgresql/root/usr/share/cont-docs/postgresql/HOME.txt

```diff
@@ -0,0 +1,86 @@
+=====================================
+Help page for PostgreSQL in container
+=====================================
+
+General documentation for PostgreSQL container usage.  Most of this
+documentation describes only proof-of-concept implemented ideas.  Be careful not
+to rely described API too much because it is still expected to change very fast.
+
+Note that this is for 1.0.0 API
+(MAJOR.MINOR.FIX).  Any change in MAJOR part of version introduces backward
+incompatibility (you should take into account that interchanging MAJOR container
+with MAJOR+N container will need some manual changes in your tooling).  On the
+other hand, incrementing MINOR version just adds some new features.
+
+
+Basic concepts
+--------------
+We try to push you to mount PostgreSQL datadir from host machine (see
```

Comment:
https://github.com/sclorg/rhscl2dockerfile/issues/23
Except for debugging, we haven't found any use-case for database container not having explicit
volume mount for data -- and it is much worse to start database server mistakenly without external data
directory.